app/scripts/modules/ecs/src/serverGroup/configure/wizard/networking/networkingSelector.component.html

```diff
@@ -94,62 +94,4 @@
       </div>
     </div>
   </div>
-
-  <div class="col-md-12" ng-if="$ctrl.command.viewState.dirty.targetGroup">
-    <div class="alert alert-warning">
-      <p>
-        <i class="fa fa-exclamation-triangle"></i>
-        The following target groups could not be found in the selected account/region/VPC and were removed:
-      </p>
-      <ul>
-        <li ng-repeat="targetGroup in $ctrl.command.viewState.dirty.targetGroup">{{ targetGroup }}</li>
-      </ul>
-      <p class="text-right">
-        <a
-          class="btn btn-sm btn-default dirty-flag-dismiss"
-          href
-          ng-click="$ctrl.command.viewState.dirty.targetGroup = null"
-          >Okay</a
-        >
-      </p>
-    </div>
-  </div>
-  <div class="form-group">
-    <div class="col-md-4 sm-label-right">
-      <b>Target Group</b>
-      <help-field key="ecs.loadBalancer.targetGroup"></help-field>
-    </div>
-    <div class="col-md-6">
-      <div class="form-control-static" ng-if="!$ctrl.command.backingData.filtered.targetGroups.length">
-        No target groups found in the selected account/region/VPC
```

Comment:
Should we carry forward this logic and not show the target group mapping component if there are no target groups?

Reply:
The target group mapping component will always contain the Target Group, and Target Group Port. If one does not exist, it will be empty with an 'Add Target Group Mappings' button, but I think we should always show the section so that users can add new ones.

Reply:
If targetGroupsAvailable is empty, won't the target group mappings row dropdown be empty?  That's the scenario where we might want to indicate that there are no target groups in selected VPC, and the user needs to select a different subnet to get some target groups (or go create some in that VPC)

Reply:
It will, should we provide an error message if none are present, or rely on the user to figure it out since there are none in the drop down?

Reply:
I think users won't understand why there are none in the dropdown, or what action they can take to populate the dropdown (choose a diff VPC), so I think the error message we previously had was useful.

Reply:
Addressed. If target groups are not available, the Add New Target Group Mappings turns into an alert, and prevents the user from adding more mappings until some are available.